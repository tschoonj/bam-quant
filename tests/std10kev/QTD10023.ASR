File: C:\xrfdaten\2014KW30\chris\std10kev\std10023.spe
Start: 21.07.2014 15:55:24
Stop: 21.07.2014 15:57:04
BESSY: 298.5914186 298.5845417
IONI: 1.463533E-10 1.455952E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 84491
Run: 100000
Dead: 16
ICR: 48500
OCR: 0
Accumulated BESSY: 59718.907904
Accumulated Ioni: 2.92126927E-08
N Accumulation: 200
Normfactor:       81030395.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: V
exp_x 92.0204010009766 92.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  23   1      4.950   3509704.      1874.     160.86
