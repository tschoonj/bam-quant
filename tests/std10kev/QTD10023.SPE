$DATA:
0 2000
19738
12771
5566
1748
311
44
9
6
2
4
0
2
1
0
1
2
153
975
1225
1155
1108
1075
1053
986
959
981
943
883
904
840
773
865
826
812
777
716
729
719
769
713
768
642
731
685
693
658
595
669
668
593
611
647
640
646
631
620
652
587
597
601
604
665
613
587
548
584
580
611
569
548
546
557
545
612
565
556
562
507
512
536
557
547
473
529
494
523
536
494
551
528
518
565
545
532
514
587
527
616
570
617
640
597
650
652
633
581
651
570
597
558
584
568
606
555
579
582
564
566
540
577
556
603
544
564
551
614
585
622
614
649
703
674
617
590
602
586
562
598
585
604
693
657
700
855
973
1108
1237
1239
1153
963
891
858
869
1062
1437
1933
2661
3448
3874
3869
3388
2537
1802
1282
913
745
706
649
585
651
663
666
670
701
654
716
666
797
810
961
1024
1152
1240
1232
1119
956
912
789
776
753
758
776
836
787
839
853
844
858
887
896
902
1008
975
949
943
1044
1074
1023
1105
1063
1172
1136
1170
1236
1233
1147
1192
1182
1251
1244
1250
1168
1224
1416
1338
1457
1553
1666
1749
1919
2010
2347
2650
3030
3720
5030
7531
13769
27652
55650
105861
183270
282396
386026
465741
495205
463672
381636
277449
176744
100516
50406
23050
9693
4412
2351
1878
1839
2089
2449
3446
5141
8635
14213
23288
35215
48437
61255
68341
67866
60636
47978
33727
21112
11853
5950
2611
1193
455
176
94
71
48
42
44
48
41
38
39
39
34
34
36
38
38
28
36
28
22
17
23
24
16
20
11
19
20
15
17
19
23
37
52
45
86
104
116
122
94
94
77
47
29
36
13
11
9
9
10
8
4
7
14
12
11
8
17
19
9
9
14
15
20
5
14
21
18
26
26
23
13
27
24
11
22
12
10
8
8
9
7
10
9
5
11
16
18
16
18
29
25
23
20
22
16
11
8
10
11
2
9
9
6
10
10
10
6
16
20
11
18
21
19
29
36
46
50
35
58
63
58
65
46
52
44
44
28
30
29
25
17
10
6
13
13
13
5
7
15
8
16
19
23
49
33
56
47
51
65
58
38
25
30
11
8
17
20
14
18
16
16
9
13
18
14
12
16
8
11
12
9
14
10
11
15
12
18
22
17
24
14
14
18
21
19
14
16
20
26
36
23
21
42
36
30
43
43
60
69
102
125
194
266
406
598
795
1065
1330
1526
1769
1846
1801
1633
1439
1169
936
701
573
417
292
234
172
116
97
73
98
102
161
200
274
325
366
388
393
431
334
338
224
188
127
87
46
38
12
6
7
6
5
4
4
8
15
15
21
24
21
26
28
32
30
14
16
17
7
5
4
4
0
0
1
0
0
0
1
0
1
0
0
0
1
1
1
3
0
0
2
0
0
1
0
0
0
0
0
1
0
0
0
0
2
0
0
2
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
1
0
0
0
0
1
1
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
1
0
0
1
1
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
2
0
0
2
1
1
10
2
0
1
4
8
4
2
5
3
9
6
1
3
3
4
2
1
4
2
0
0
1
1
1
1
1
1
0
2
0
2
1
1
2
2
1
1
2
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std10kev\std10023.spe
Start: 21.07.2014 15:55:24
Stop: 21.07.2014 15:57:04
BESSY: 298.5914186 298.5845417
IONI: 1.463533E-10 1.455952E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 84491
Run: 100000
Dead: 16
ICR: 48500
OCR: 0
Accumulated BESSY: 59718.907904
Accumulated Ioni: 2.92126927E-08
N Accumulation: 200
Normfactor:       81030395.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: V
 
exp_x 92.0204010009766 92.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  23   1      4.950   3509704.      1874.     160.86
