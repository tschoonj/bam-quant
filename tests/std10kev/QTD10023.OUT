     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:26:57  บ   
     บ  Spectrum: QTD10023.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 - 500        4 iterations done    บ   
     บ  ChiSquare =   243.6   last change =   .04%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -30.3 ๑   100.0   บ     -30.3 ๑      .6   บ   
     บ  GAIN (eV/ch)  บ     20.251 ๑   2.000   บ    20.251 ๑    .002   บ   
     บ det NOISE (eV) บ      -73.5 ๑    40.0   บ     -73.5 ๑    -3.5   บ   
     บ  FANO factor   บ       .229 ๑    .050   บ      .229 ๑    .003   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 V -Ka                           3509704. ๑   1874.           บ   
     บ    KA1         4.950   1.00000     3509704. ๑   1874.    160.9  บ   
     บ              245.929    137.15        1365.           8.31E-01  บ   
     บ                                                                 บ   
     บ  2 V -Kb                            518970. ๑    722.           บ   
     บ    KB1         5.427   1.00000      518970. ๑    722.    105.3  บ   
     บ              269.484    145.41         928.           8.69E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           23625. ๑    180.           บ   
     บ    KA1         9.900    .77158       18229. ๑    139.     45.2  บ   
     บ              490.363    207.43         136.           9.76E-01  บ   
     บ    KB1        10.377    .22842        5396. ๑     41.    262.9  บ   
     บ              513.917    212.99           0.           9.79E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
